terraform/shared/shared.tf.json.template.py

```diff
@@ -1014,6 +1014,29 @@ def conformance_pack(name: str) -> str:
                 ]
             }
         },
+        'aws_wafv2_regex_pattern_set': {
+            config.blocked_user_agents_regex_term: {
+                'name': config.qualified_resource_name(config.blocked_user_agents_regex_term),
+                'scope': 'REGIONAL',
+                'regular_expression': [
+                    {
+                        'regex_string': 'UAEmdDd991fowvU7Kc-RVc3LUfU7cmcjwBg'
+                    },
+                    {
+                        'regex_string': 'aria2'
```

Comment:
```suggestion
                        'regex_string': 'aria2'
```

This would have to be anchored at the beginning and include the `/` that's between the name and the version.